provisioners.tf

```diff
@@ -49,6 +49,19 @@ resource "helm_release" "karpenter_provisioners" {
 
   depends_on = [
     helm_release.karpenter,
-    module.cluster.aws_auth_configmap_yaml
+  ]
+}
+
+resource "null_resource" "wait_for_scaledown" {
+  count = local.cluster.install && local.config_karpenter.install ? 1 : 0
+
+  provisioner "local-exec" {
+    when        = destroy
+    interpreter = ["bash", "-c"]
+    command     = "while [[ $(kubectl get nodes --context test -l karpenter.sh/provisioner-name -o json | jq '.items | length') != 0 ]]; do sleep 5; done; sleep 3;"
```

Comment:
This is assuming the context name. We need to find a way to set up the context w/ token or otherwise. 🤔